website/docs/r/qumulo_file_system.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+subcategory: "Qumulo"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_qumulo_file_system"
+description: |-
+  Manages an Azure Native Qumulo Scalable File System.
+---
+
+# azurerm_qumulo_file_system
+
+Manages an Azure Native Qumulo Scalable File System.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_virtual_network" "example" {
+  name                = "example-vnet"
+  address_space       = ["10.0.0.0/16"]
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_subnet" "example" {
+  name                 = "example-subnet"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes     = ["10.0.1.0/24"]
+
+  delegation {
+    name = "delegation"
+    service_delegation {
+      actions = ["Microsoft.Network/virtualNetworks/subnets/join/action"]
+      name    = "Qumulo.Storage/fileSystems"
+    }
+  }
+}
+
+resource "azurerm_qumulo_file_system" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  admin_password      = ")^X#ZX#JRyIY}t9"
+  availability_zone   = "1"
+  delegated_subnet_id = azurerm_subnet.example.id
+  plan_id             = "azure-native-qumulo-v3"
+  storage_sku         = "Standard"
+  email               = "test@test.com"
+  tags = {
+    environment = "test"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of this Azure Native Qumulo Scalable File System resource. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) Specifies the name of the Resource Group within which this Azure Native Qumulo Scalable File System should exist. Changing this forces a new resource to be created.
+
+* `location` - (Required) The Azure Region where the Azure Native Qumulo Scalable File System should exist. Changing this forces a new resource to be created.
+
+* `admin_password` - (Required) Initial administrator password of the Azure Native Qumulo Scalable File System. Changing this forces a new resource to be created.
+
+* `plan_id` - (Required) Specifies the marketplace plan ID. Changing this forces a new resource to be created.
+
+* `storage_sku` - (Required) The storage Sku. Possible values are `Cold_LRS`, `Hot_LRS` and `Hot_ZRS`. Changing this forces a new resource to be created.
+
+* `subnet_id` - (Required) Delegated subnet ID for Vnet injection. Changing this forces a new resource to be created.
+
+* `email` - (Required) The email address. Changing this forces a new resource to be created.
```

Comment:
Could we provide more information here, what is this email for? This also needs to be listed under `admin_password` per our [ordering guidelines](https://github.com/hashicorp/terraform-provider-azurerm/blob/9e3693b540db0c53d2565b3892db5ab8d011751e/contributing/topics/guide-new-resource.md?plain=1#L169-L175).

Reply:
updated, changed to `The email address used for the Azure Native Qumulo Scalable File System`.